ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/SandboxConfig.scala

```diff
@@ -66,5 +74,10 @@ object SandboxConfig {
     )
 
   lazy val default: SandboxConfig =
-    nextDefault.copy(seeding = None)
+    nextDefault.copy(
+      partyConfig = nextDefault.partyConfig.copy(
+        implicitPartyAllocation = false // unnecessary: parties are always allocated implicitly
```

Comment:
Hm, I don't understand the comment. Why not set it to `true` then?

Reply:
Because it effectively does it twice, which is unnecessary. I'll update the comment.

Reply:
Wrote more commentary, about both this and why you should be careful setting `implicitPartyAllocation = true` in general.